hstrat/_auxiliary_lib/_lazy_attach.py

```diff
@@ -0,0 +1,87 @@
+from typing import Any, Callable, Dict, List, Optional, Tuple
+
+from lazy_loader import attach
+from opytional import apply_if
+
+
+def lazy_attach(
```

Comment:
Could we consolidate `lazy_attach` and `lazy_attach_stub` into one function and use a kwarg having to do with importing all attributes from subpackage control that behavior?

Reply:
I don't think so, as the functions operate in a different enough manner that combining them might take too much effort and be awkward to call